src/main/resources/db/test/migration/V900__case_note_types_data.sql

```diff
@@ -28,3 +28,7 @@ VALUES ('OMIC', 'GEN', 'General OMIC Note', true, now(), 'SYSTEM');
 INSERT INTO CASE_NOTE_TYPE (PARENT_TYPE, SUB_TYPE, DESCRIPTION, ACTIVE, CREATE_DATE_TIME, CREATE_USER_ID)
 VALUES ('OMIC', 'SPECIAL', 'Special OMIC Note', true, now(), 'SYSTEM');
 
+INSERT INTO CASE_NOTE_TYPE (PARENT_TYPE, SUB_TYPE, DESCRIPTION, ACTIVE, SENSITIVE, CREATE_DATE_TIME, CREATE_USER_ID)
+VALUES ('OMIC', 'OMIC_OPEN', 'Open Case Note', true, false, now(), 'SYSTEM');
```

Comment:
wondering if this should be in the general rather than the test flyway or aren't they ready for the open case note yet?

Reply:
Not ready yet